### ISSUE TITLE
release builds 

### ISSUE THREAD

**Issue by achiko** [PolymathNetwork/Polymesh#589]
It would be better if you provide release builds for different versions  to run nodes.  It's hard to follow every new version and build from scratch. 
 I juts need to interact with Polymesh Network. 
Thank you.

**Comment by maxsam4** [comment]
@achiko Thanks for reaching out. We are currently setting up the required pipelines and we'll soon start publishing artifacts/binaries for all our releases.

**Comment by maxsam4** [comment]
Release binaries will now be automatically published on https://github.com/PolymathNetwork/Polymesh/releases
